scripts/joysetup.sh

```diff
@@ -755,6 +755,8 @@ setup_datasets()
             # Linux mounts to /var
             zfs set mountpoint=/var "${VARDS}" || \
             fatal "failed to set the mountpoint for ${VARDS}"
+            # Restart triton-lxd service so imgadm list succeeds later
```

Comment:
Change this comment to:

    # Restart triton-lxd service now that /var is mounted properly